.github/CODEOWNERS

```diff
@@ -11,6 +11,7 @@
 /.cargo                             @benesch
 /.config                            @benesch
 /.github                            @benesch
+/bazel                              @parkmycar @guswynn @maddyblue
```

Comment:
Sorry, didn't see the original PRs go past, but if Bazel doesn't care, can we put this somewhere besides the root of the repo? This is what `misc` was made for!

Reply:
Totally, updated!